src/network/clientpackethandler.cpp

```diff
@@ -410,10 +410,10 @@ void Client::handleCommand_Movement(NetworkPacket* pkt)
 	Player *player = m_env.getLocalPlayer();
 	assert(player != NULL);
 
-	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g;
+	float mad, maa, maf, msw, mscr, msf, mscl, msj, lf, lfs, ls, g, dmg;
 
 	*pkt >> mad >> maa >> maf >> msw >> mscr >> msf >> mscl >> msj
-		>> lf >> lfs >> ls >> g;
+		>> lf >> lfs >> ls >> g >> dmg;
```

Comment:
Why is it "not allowed" to modify an existing protocol version in a backwards compatible way? I mean, it will only affect people that haven't updated and run on the master branch.

And to your question HybridDog: Perhaps we need a new packet for this, as only the server knows the used protocol...